requirements.txt

```diff
@@ -16,3 +17,4 @@ pytest-asyncio
 google-cloud-storage
 
 protobuf==3.20.1
+ipython
```

Comment:
Maybe this could also go from the non-dev requirements

Reply:
No, we need it for the Shapley values